### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by aldemeery**
### Added

-   Player class.
-   Evaluator class.
-   Board class.
-   Game class.
-   Game logic.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    const announcementNode = document.querySelector('#announcement');
+    announcementNode.style.display = 'block';
+    announcementNode.textContent = `${this.currentPlayer.name} wins!`;
+  }
+
+  function displayDraw() {
+    const announcementNode = document.querySelector('#announcement');
+    announcementNode.style.display = 'block';
+    announcementNode.textContent = 'Draw!';
+  }
```
Comment (bellom) [inline src/index.js:168]: Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+          }
+        }
+      });
+    });
+
+    const settingBtn = document.querySelector('#settings');
+    settingBtn.addEventListener('click', () => {
+      const modal = document.querySelector('#modal');
+      modal.style.display = 'block';
+    });
```
Comment (bellom) [inline src/index.js:200]: Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    const p2NameNode = document.querySelector('#p2');
+    const p1ScoreNode = document.querySelector('#p1score');
+    const p2ScoreNode = document.querySelector('#p2score');
+
+    p1NameNode.textContent = this.playerX.name;
+    p1ScoreNode.textContent = this.playerX.score;
+
+    p2NameNode.textContent = this.playerO.name;
+    p2ScoreNode.textContent = this.playerO.score;
+  };
```
Comment (bellom) [inline src/index.js:244]: Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @aldemeery  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/index.js (hunk 1 above)
Comment (aldemeery) [reply src/index.js:168]: Resolved

File: src/index.js (hunk 2 above)
Comment (aldemeery) [reply src/index.js:200]: Resolved

File: src/index.js (hunk 3 above)
Comment (aldemeery) [reply src/index.js:244]: Resolved

**Comment by aldemeery** [comment]
All issues have been resolved. DOM code has been moved into a separate module.

**Review by bellom — APPROVED** [review]
## Hi @aldemeery :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**